pytorch_lightning/utilities/seed.py

```diff
@@ -41,7 +41,10 @@ def seed_everything(seed: Optional[int] = None) -> int:
 
     try:
         if seed is None:
-            seed = os.environ.get("PL_GLOBAL_SEED", _select_seed_randomly(min_seed_value, max_seed_value))
+            if "PL_GLOBAL_SEED" in os.environ:
+                seed = os.environ["PL_GLOBAL_SEED"]
+            else:
+                seed = _select_seed_randomly(min_seed_value, max_seed_value)
```

Comment:
btw, why if you call seed, and you want random, it raises a warning, the `_select_seed_randomly` always raises warning but shall be only if seed is a wrong range...